WTWSMS/history/characters/WTWSMS_enlil.txt

```diff
@@ -6229,6 +6229,12 @@
 	573.1.1 = {
 		birth = yes
 	}
+	610.1.1 = {
+	    trait = islamic
```

Comment:
Islamic is not a trait.

Reply:
I know, its just mistake, i fixed this now.